src/helpers/validate.ts

```diff
@@ -14,10 +14,15 @@ export function validateURL(url: string): boolean {
   return validator.isURL(url, { require_protocol: true })
 }
 
-export function validateFlags(flags: string): boolean {
+export function validateFlags(flag: string): boolean {
```

Comment:
```suggestion
export function validateFlags(flags: string[]): void {
```

Reply:
This is being passed a single flag as part of a filter command, I believe, @mitchell-codecov